builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -196,18 +211,29 @@ func respondReadIssuer(issuer *issuerEntry) (*logical.Response, error) {
 		revSigAlgStr = ""
 	}
 
+	data := map[string]interface{}{
+		"issuer_id":                      issuer.ID,
+		"issuer_name":                    issuer.Name,
+		"key_id":                         issuer.KeyID,
+		"certificate":                    issuer.Certificate,
+		"manual_chain":                   respManualChain,
+		"ca_chain":                       issuer.CAChain,
+		"leaf_not_after_behavior":        issuer.LeafNotAfterBehavior.String(),
+		"usage":                          issuer.Usage.Names(),
+		"revocation_signature_algorithm": revSigAlgStr,
+		"issuing_certificates":           []string{},
+		"crl_distribution_points":        []string{},
+		"ocsp_servers":                   []string{},
+	}
+
+	if issuer.AIAURIs != nil {
```

Comment:
I'm a bit conflicted about this, but I think the right thing to do here would be to always put the key in the response and leave it blank?

Reply:
Hmm, I was kinda weirded by that. In the old days, we wouldn't return a config object at all (nil body) if it had no config. Now we'll return an empty config with no data... Since the objects are generally flat, we leave the top-level keys with empty values.

So I think I agree we should leave the top-level key but leave it nil perhaps? Should we populate the keys of the map too? Hmmm....

Reply:
Oh never mind, these are top-level keys. D'oh. Fixing...

Reply:
Wait, I'm really confused by this comment. We always place the key in the response. So this doesn't need to be fixed, it is correct as-is.

Reply:
Ah your right, I missed the blank keys added to the map above. My bad, sorry for the confusion...

Reply:
No problem!